.changeset/smooth-radios-wink.md

```diff
@@ -0,0 +1,8 @@
+---
+"@logto/console": patch
+---
+
+fix display logic for "Always issue refresh token" and "Refresh token TTL" according to OIDC configuration
```

Comment:
It's nothing about "Always issue refresh token".
The change actually impacts "Rotate refresh token" toggle switch and "Refresh token TTL" field

Reply:
i was unconscious